RELEASE/scripts/autoscend/paths/i_love_u_hate.ash

```diff
@@ -11,7 +11,7 @@ boolean iluh_foodConsumable(string str)
 	}
 
 	//Not actually going to ever be consumed but need this exception to actually make it for the Palindome quest
-	if(contains_text(str.to_lower_case(), "stunt nut"))
```

Comment:
I'll approve as this works. I do have 2 optional improvements
1. make a variable for `str.to_lower_case()` since it is used often in this function
2. have this check each specific food - wet stew, stunt nuts, or wet stunt nut stew. What you have obviously works, but I think this would be more readable as allowing wet stunt nut stew is a little hidden atm

up to you if you want to make this changes!